tuna/worker_interface.py

```diff
@@ -338,66 +367,71 @@ def callback():
 
       assert session_retry(session, callback, lambda x: x(), self.logger)
 
-  def exec_command(self, cmd):
+  def exec_command(self, cmd: str) -> Tuple[int, StringIO, StringIO]:
```

Comment:
Would be good if the returns from this value were of consistent type with exec_docker_cmd.

Reply:
When we normalize the return parameters of exec_docker_cmd with exec_command, we get an error on _StringIO object while processing string. As _StringIO doesn't support translate() cmd( in sqlalchemy lib). get_rocm_v->exe_docker_cmd-> returns error, when we set the version parameter as StringIO. As of now, We need to define the return parameters of exec_docker_cmd as Tuple[int, str, StringIO].

Reply:
It would do just as well to match this functions output types to exec_docker_cmd, eg use Tuple[int, str, StringIO] and match the string parsing functionality of the other function.